.env.example

```diff
@@ -1 +1,2 @@
-PREZLY_ACCESS_TOKEN=
\ No newline at end of file
+PREZLY_ACCESS_TOKEN=
+PREZLY_NEWSROOM_ID=
```

Comment:
Should this be called `PREZLY_NEWSROOM_UUID` ?

Reply:
Yes, please. `NEWSROOM_UUID`.